src/main/java/org/gogame/server/domain/entities/LeaderboardEntity.java

```diff
@@ -14,6 +14,7 @@
 public class LeaderboardEntity {
 
     @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE)
     @Column(name = "user_pos", nullable = false, unique = true)
     private Long userPos;
```

Comment:
isn't this the user's position on the leaderboard? why is it autogenerated?

Reply:
`userPos` is completely screwed up and should be removed. Thanks for issuing that